eng/pipelines/templates/steps/test.yml

```diff
@@ -9,42 +9,46 @@ steps:
     parameters:
       AgentImage: ${{ parameters.OSName}}
 
+  - template: /eng/common/pipelines/templates/steps/save-package-properties.yml
+    parameters:
+      ServiceDirectory: ${{parameters.ServiceDirectory}}
+
   - script: |
       node common/scripts/install-run-rush.js install
     displayName: "Install dependencies"
 
   - template: /eng/pipelines/templates/steps/set-artifact-packages.yml
     parameters:
-      Artifacts: ${{ parameters.Artifacts }}
+      PackageInfo: $(Build.ArtifactStagingDirectory)/PackageInfo
```

Comment:
I assume we are intentionally excluding Artifacts here because we are just taking what we have a PackageInfo for and not filtering based on the artifact list?

Reply:
This one should definitely still pass `Artifacts`. The way that `js - pullrequest` gets the correct set is that it DOESN'T pass an artifact list, so the filtering of the discovered packages doesn't happen. Thanks for the catch.